pyigm/cgm/galaxy.py

```diff
@@ -0,0 +1,210 @@
+"""  Module for CGM of the Milky Way
+"""
+
+from __future__ import print_function, absolute_import, division, unicode_literals
+
+import numpy as np
+import pdb
+import warnings
+
+from astropy import units as u
+from astropy import constants as const
+from astropy.table import Table, Column
+from astropy.coordinates import SkyCoord
+
+
+from linetools.spectralline import AbsLine
+from linetools.isgm.abscomponent import AbsComponent
+
+from pyigm.abssys.igmsys import IGMSystem
+from pyigm.field.galaxy import Galaxy
+from .cgm import CGM, CGMAbsSys
+from .cgmsurvey import CGMAbsSurvey
+import pyigm
+
+c_kms = const.c.to('km/s').value
+
+class GalaxyCGM(CGM):
+    """Inherits CGM Abs Survey
+
+    Parameters:
+    -----------
+    fits_path : str, optional
+      Path to the FITS data files for COS-Halos
+    cdir : str, optional
```

Comment:
Surely these wouldn't need point to the COS-Halos data for this object, right? The CGM base class doesn't require such paths.